pkg/minikube/service/service.go

```diff
@@ -265,22 +264,24 @@ func PrintServiceList(writer io.Writer, data [][]string) {
 	table.Render()
 }
 
-// WaitAndMaybeOpenService waits for a service, and opens it when running
-func WaitAndMaybeOpenService(api libmachine.API, namespace string, service string, urlTemplate *template.Template, urlMode bool, https bool,
-	wait int, interval int) error {
+// WaitForService waits for a service, and opens it when running
+func WaitForService(api libmachine.API, namespace string, service string, urlTemplate *template.Template, urlMode bool, https bool,
+	wait int, interval int) (string, error) {
+
+	var urlString string
```

Comment:
Just `url` is fine as a variable name. It generally isn't useful to repeat the type of an object in the object name.

See also https://github.com/golang/go/wiki/CodeReviewComments#variable-names